sites/kit.svelte.dev/src/lib/search/SearchBox.svelte

```diff
@@ -106,7 +106,7 @@
 
 <svelte:window
 	on:keydown={(e) => {
-		if (e.code === 'KeyK' && (navigator.platform === 'MacIntel' ? e.metaKey : e.ctrlKey)) {
+		if (e.key === 'k' && (navigator.platform === 'MacIntel' ? e.metaKey : e.ctrlKey)) {
```

Comment:
edge case, but do we need to consider the situation where caps lock is on?

```suggestion
		if ((e.key === 'k' || e.key === 'K') && (navigator.platform === 'MacIntel' ? e.metaKey : e.ctrlKey)) {
```

Reply:
There is no need, I just try it and keeps working.

Reply:
oh, fun — i tried on https://keycode.info/ and it shows up as `K` with caps lock (hence my comment) but if you're also holding the cmd key down it's `k`. Keyboard events are endlessly weird!